easy_my_coop_connector/models/subscription_request.py

```diff
@@ -15,7 +15,7 @@
 class SubscriptionRequest(models.Model):
     _inherit = "subscription.request"
 
-    source = fields.Selection(selection_add=[("emc_api", "Easy My Coop API")])
+    source = fields.Selection(selection_add=[("cooperator_api", "Easy My Coop API")])
```

Comment:
This line already exists in cooperator_api_logs. Why is it here? Should it be removed?

Reply:
It's in `cooperator_api` not in `cooperator_api_logs`.

`cooperator_api` and `cooperator_connector` should not be installed at the same time : do not fix `easy_my_coop_connector` at this time. 

Although you have a point that we may want to have a different field name for
- `cooperator_api` : tracks the requests created through a POST on `api/subscription-request`
- `easy_my_coop_connector` : tracks requests pulled from `cooperator_api` module.

Reply:
> do not fix easy_my_coop_connector at this time.

Something will need to be fixed, because the CI installs both `cooperator_api` and `easy_my_coop_connector` simultaneously.

Solutions are:

1. Mark these modules as conflicting in the CI and run them separately. Don't change anything in `easy_my_coop_connector`.
2. Rename the field `source` in `easy_my_coop_connector` to something else.
3. Rename the field `source` in `cooperator_api` to something else.

(Or a combination of the above).

Reply:
In any case, let's do 1 in both the CI and in the module mannifest.

`source` should not be renamed as it is used elsewhere. let's rather change the selection to
- `cooperator_api` or just `api` in `cooperator_api`
-  `platform` or `external-api` or `?` in `easy_my_coop_connector`

We struggled with @huguesdk : suggestions welcome.

Reply:
> in the module manifest

I can't find a key to specify incompatibility in the module manifest. Done for CI though.

> platform or external-api

Opted for `connector_api`.